docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,499 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d299c1e8",
+   "metadata": {},
+   "source": [
+    "# Finding Label Errors in Object Detection (object detection) Datasets\n",
```

Comment:
```suggestion
    "# Finding Label Errors in Object Detection Datasets\n",
```

Reply:
unsure if I'm missing something for why there is both upper and lower case name in the title?